drivers/hmis/spec/requests/hmis/organization_spec.rb

```diff
@@ -75,5 +75,19 @@
       orgs = result.dig('data', 'organizations', 'nodes')
       expect(orgs.length).to eq(0)
     end
+
+    context 'when there are multiple HMIS data sources' do
+      let!(:ds2) { create :hmis_data_source }
+      let!(:o5) { create :hmis_hud_organization, data_source: ds2, user: u1, OrganizationName: 'Pineapples' }
+      let!(:ds2_access_control) { create_access_control(hmis_user, ds2) }
+
+      it 'does not return organizations from other data source' do
+        # expect(Hmis::Hud::Organization).to receive(:viewable_by).and_return(Hmis::Hud::Organization.all)
```

Comment:
Ok, here's a thread to talk about the original todo you left me with, which is testing the graphql permission checker against organization and data source. 

This line is commented out because, if I add it, I would expect the graphql query to fail. Instead, what happens is that the query succeeds and returns 5 orgs, including the one in ds2. 

I think, maybe, it is because when the graphql permission checker's `current_permission_for_context?` is called on a client, enrollment, or project, it is passed with a specific permission, whereas when it's called on base_field, it is passed without a specific permission. Inside base_field's `authorized?` method, you can see that it is only checking `current_permission_for_context?` if `permissions.any?`. I haven't fully followed what this means, but I think it implies that the recent changes we made in `current_permission_for_context` only impact entities where the access loader is project, client, or enrollment.

Reply:
Okay. Sorry I left you with a vague todo.
Let's huddle, but here are some initial notes:
* there is no permission for org visibility, it's purely based on access control "collections". If you wanted to test organization-level permissions that would be either `can_edit_organization` (evaluated by SubmitForm mutation) and `can_delete_organization` (evaluated by DeleteOrganization)
* there is no object-level authorization on the organization type
* field-level authorization is different from object-level authorization. I'm not totally tracking on yoru read of

Reply:
Sum up from huddle: This was just my misunderstanding of `base_field`, which is for field level permissions (as opposed to object level permissions).